src/stores/standalone/firewall.ts

```diff
@@ -81,6 +81,7 @@ export interface NatRule {
   dest_ip: string
   target: NatRuleAction
   snat_ip?: string
+  device: string
```

Comment:
Since `device` can be an empty string I'd consider it as an optional attribute, but it's just a matter of taste, we can leave it as required

Reply:
The field is not optional, it's empty, if we check for it's nullability we don't actually check for anything